packages/web-embed-api/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.2.1
+
+- Use `{clientLabel}.formsort.app` if no origin was specified. `flow.formsort.com/{clientLabel}` is deprecated.
```

Comment:
Revert this change please, the changelog should be auto-generated.

Reply:
What will it be based on?